templates/assets.phtml

```diff
@@ -117,3 +120,17 @@
 <?php include("_pagination.phtml") ?>
 <a href="<?php echo raw($basepath) . '/asset/edit?' . raw($query) . '&asset=-1' ?>" class="btn btn-warning"><i class="glyphicon glyphicon-hourglass"></i> View Pending</a>
 <?php include("_footer.phtml") ?>
+<script>
+  var search_options = document.getElementById('search_options');
+  function show_hide_search_options() {
+    if (search_options.style.display == 'none') {
```

Comment:
Changing the display options should be done in CSS using [media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries).